plaso/parsers/sqlite_plugins/macos_tcc.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+"""Parser for Apple's Transaprency, Consent, Control (TCC) framework database.
+
+The TCC data is stored in SQLite databases at the following locations:
+* /Library/Application Support/com.apple.TCC/TCC.db
+* %homedir%/Library/Application Support/com.apple.TCC/TCC.db
+"""
+
+from __future__ import unicode_literals
+
+from dfdatetime import posix_time as dfdatetime_posix_time
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.parsers import sqlite
+from plaso.parsers.sqlite_plugins import interface
+
+
+class MacOSTCCEntry(events.EventData):
+  """macOS TCC event data.
+
+  Attributes:
+    allowed (bool): whether access to the service was allowed.
+    client (str): name of the client requesting access to the service.
+    prompt_count (int): number of times an appplication prompted for access
```

Comment:
I assume this prompts the user? Let's mention that in the docstring.

Reply:
Fixed.